src/sentry/sentry_metrics/client/__init__.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+
+from sentry.utils.services import LazyServiceWrapper
+
+from .base import GenericMetricsBackend
+
+backend = LazyServiceWrapper(
+    GenericMetricsBackend,
+    settings.SENTRY_METRICS_INTERFACE_BACKEND,
+    settings.SENTRY_METRICS_INTERFACE_BACKEND_OPTIONS,
+)
+backend.expose(locals())
```

Comment:
i'd like to stop using `expose` as it is not typesafe. can we make people use methods on `backend` directly?